### PR TITLE
Squash Dependabot Updates

### PR DESCRIPTION
**Pull request by StormFireFox1**
Dependabot had a bunch of security updates in our repo. Squash all updates in one single PR.

These are all security updates; no significant change in code should be required, such as for the dependencies listed in #547.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -8090,9 +8090,15 @@ minimist-options@4.1.0:
...
 minimist@^1.1.1, minimist@^1.1.3, minimist@^1.2.0, minimist@^1.2.5:
+<<<<<<< Updated upstream
   version "1.2.5"
   resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.5.tgz#67d66014b66a6a8aaa0c083c5fd58df4e4e97602"
   integrity sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==
+=======
+  version "1.2.6"
+  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.6.tgz#8637a5b759ea0d6e98702cfb3a9283323c93af44"
+  integrity sha512-Jsjnk4bw3YJqYzbdyBiNsPWHPfO++UGG749Cxs6peCu5Xg4nrena6OVxOYxrQTqww0Jmwt+Ref8rggumkTLz9Q==
+>>>>>>> Stashed changes
```
Comment (shravanhariharan2) [inline yarn.lock:8102]: looks like there's some git merge changes here

File: yarn.lock [hunk 2]
```diff
@@ -8133,10 +8139,17 @@ mkdirp@^0.5.1, mkdirp@^0.5.5, mkdirp@~0.5.1:
...
 moment@2.x, moment@^2.24.0, moment@^2.29.1:
   version "2.29.1"
   resolved "https://registry.yarnpkg.com/moment/-/moment-2.29.1.tgz#b2be769fa31940be9eeea6469c075e35006fa3d3"
   integrity sha512-kHmoybcPV8Sqy59DwNDY3Jefr64lK/by/da0ViFcuA4DH0vQg5Q6Ze5VimxkfQNSC+Mls/Kx53s7TjP1RhFEDQ==
+=======
+moment@2.x, moment@^2.24.0, moment@^2.29.2:
+  version "2.29.2"
+  resolved "https://registry.yarnpkg.com/moment/-/moment-2.29.2.tgz#00910c60b20843bcba52d37d58c628b47b1f20e4"
+  integrity sha512-UgzG4rvxYpN15jgCmVJwac49h9ly9NurikMWGPdVxm8GZD6XjkKPxDTjQQ43gtGgnV3X0cAyWDdP2Wexoquifg==
+>>>>>>> Stashed changes
```
Comment (shravanhariharan2) [inline yarn.lock:8153]: here as well

**Review by shravanhariharan2 — COMMENTED** [review]
tested the deploy preview and nothing seemed to break (I was able to view leaderboard / order merch / view events etc. without any bugs)

File: yarn.lock (hunk 1 above)
Comment (StormFireFox1) [reply yarn.lock:8102]: Yeah, I'll merge them right now.

**Review by steets250 — APPROVED** [review]
LGTM 🚀

**Pull request merged**